openwebif/api.py

```diff
@@ -25,6 +25,7 @@
 URL_ABOUT = "/api/about"
 URL_TOGGLE_VOLUME_MUTE = "/web/vol?set=mute"
 URL_SET_VOLUME = "/api/vol?set=set"
+URL_MESSAGE = "/api/message?timeout={}&type=1&text={}"
```

Comment:
From the docs around type:

> type - (number) 0: YESNO, 1: INFO, 2: WARNING, 3: ERROR

It would be useful to have this as a parameter also in the `__init__` which defaulted to `1`.